medical_procedure_external/views/medical_procedure_external_request_template.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record
+        model="ir.ui.view"
+        id="medical_procedure_external_request_template_form_view"
+    >
+        <field
+            name="name"
+        >medical.procedure.external.request.template.form (in medical_procedure_external_request)</field>
+        <field name="model">medical.procedure.external.request.template</field>
+        <field name="arch" type="xml">
+            <form>
+                <header />
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <group>
+                        <field name="name" string="Name" />
+                        <field name="title" string="Title" />
+                        <field name="active" invisible="1" />
+                    </group>
+                    <notebook>
+                        <page name="composition" string="Composition">
+                            <field name="composition" />
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record
+        model="ir.ui.view"
+        id="medical_procedure_external_request_template_tree_view"
+    >
+        <field
+            name="name"
+        >medical.procedure.external.request.template.tree (in medical_procedure_external_request)</field>
+        <field name="model">medical.procedure.external.request.template</field>
+        <field name="arch" type="xml">
+            <tree>
```

Comment:
You could add decoration-warning when state it is in draft state and decoration-muted when it is cancelled